pyalgostrategypool/stochastic_crossover_v2.py

```diff
@@ -146,46 +112,45 @@ def strategy_select_instruments_for_exit(self, candle, instruments_bucket):
 
         # Add instrument in this bucket if you want to place an (exit) order for it
         # We decide whether to place an instrument in this bucket or not based on the decision making process given below in the loop
-        selected_instruments_bucket = []
+        selected_instruments = []
 
         # Add accompanying info in this bucket for that particular instrument in the bucket above
-        sideband_info_bucket = []
+        meta = []
 
         for instrument in instruments_bucket:
-            main_order = self.main_order.get(instrument)
+            main_order = self.main_order_map.get(instrument)
 
             # Compute various things and get the decision to place an (exit) order only if there is a current order is going on (main order is not empty / none)
             # Also check if order status is complete
             if main_order is not None and main_order.get_order_status() is BrokerOrderStatusConstants.COMPLETE:
 
                 # Check for crossover (decision making process)
-                action = self.get_decision(instrument, DecisionConstants.EXIT_POSITION)
+                crossover = self.get_crossover_value(instrument)
 
                 # For this strategy, we take the decision as:
                 # If order transaction type is buy and current action is sell or order transaction type is sell and current action is buy, then exit the order
-                if (action is ActionConstants.EXIT_SELL and main_order.order_transaction_type is BrokerOrderTransactionTypeConstants.SELL) or \
-                        (action is ActionConstants.EXIT_BUY and main_order.order_transaction_type is BrokerOrderTransactionTypeConstants.BUY):
+                if (crossover == 1 and main_order.order_transaction_type == "SELL") or (crossover == -1 and main_order.order_transaction_type == "BUY"):
```

Comment:
same crossover shortcut can be applied here?

Reply:
done